modules/ROOT/pages/clustering/setup/discovery.adoc

```diff
@@ -228,95 +232,182 @@ For more information, see <<clustering-discovery-k8s>>.
 . Verify that `CALL dbms.cluster.showParallelDiscoveryState()` now shows `V2_ONLY` running.
 Note that `stateComparison` is `N/A` because you do not have v1 to compare states anymore.
 
 [[discovery-v1-to-v2-new-server]]
 === New server rolling
+The new server rolling requires three running servers and three new servers.
+. For all the servers, ensure that new settings are added to the configuration as detailed the xref:clustering/setup/discovery.adoc#_preparation[Preparation] section.
++
+As an example, for those using the list resolver, the settings for all the servers should include:
++
+```
+dbms.cluster.discovery.resolver_type=LIST
 
-. Ensure that the new settings are added to the configuration as detailed in the preparation section.
-. Start up three new servers with the setting `dbms.cluster.discovery.version=V1_OVER_V2`
+dbms.cluster.discovery.endpoints=server01.example.com:5000,server02.example.com:5000,server03.example.com:5000,server04.example.com:5000,server05.example.com:5000,server06.example.com:5000
+dbms.cluster.discovery.v2.endpoints=server01.example.com:6000,server02.example.com:6000,server03.example.com:6000,server04.example.com:6000,server05.example.com:6000,server06.example.com:6000
+```
+. Start up the three new servers with the setting `dbms.cluster.discovery.version=V1_OVER_V2`.
 +
 image:discovery-service/v1_over_v2.png[]
-
-. Using `bolt://`, connect to servers 4, 5, 6, and run the following procedure.
-It is important to connect via `bolt://` because otherwise the procedure might be routed and executed not on the intended server.
++
+. Using `bolt://`, connect to the system database of servers 4, 5, 6, and run the following procedure.
+This can be done using via `./cypher-shell -a bolt://localhost:7685 -d system` for example.
 +
 [source,cypher]
 ----
 CALL dbms.cluster.showParallelDiscoveryState();
 ----
 +
-The output should display `Matching` in the `stateComparison`.
-If they are not, wait and try again.
-Once matching, servers 1, 2, and 3 can be shut down.
-
-. Start up another three new servers with the setting `dbms.cluster.discovery.version=V2_OVER_V1`.
+The expected result should display `v2ServerCount` as 3.
+`stateComparison` that does show matching is fine at this point.
++
+.Expected result
+[queryresult]
+----
+ +---------------------------------------------------------------------------------------------------------+
+ | mode         | stateComparison                                          | v1ServerCount | v2ServerCount |
+ +---------------------------------------------------------------------------------------------------------+
+ | "V1_OVER_V2" | "States are not matching after PT55M36.693S: (score:29)" | "6"           | "3"           |
+ +---------------------------------------------------------------------------------------------------------+
+----
++
+. Deallocate, drop, and shut down servers 1, 2, 3. 
+. Run `neo4j-admin unbind` on these three servers.
```

Comment:
It's not 'new server rolling' if we are re-using servers. This should be sympathetic to automatic processes which just always add a new server. So I think it's OK that it's a detailed process, that maybe has 'servers1-6' in the discovery list sometimes, sometimes 'servers4-9', etc.

Reply:
This makes alot of sense, it has been tweaked in [053952e](https://github.com/neo4j/docs-operations/pull/1756/commits/053952ee38ee5e55b30cea8e1a369ea3f4412f62)

Reply:
Very nice.